src/homework5/runner.py

```diff
@@ -0,0 +1,21 @@
+import allfunc
+
+
+def run_fnc(*args):
```

Comment:
wrong name according to task description

Reply:
fixed